Test/UnitTest1.cs

```diff
@@ -0,0 +1,70 @@
+using Training;
+
+namespace MSTestProject {
+   [TestClass]
+   public class UnitTest1 {
+
+      [TestMethod]
+      public void AddTest () {
+         myList.Add (1);
+         myList.Add (2);
+         List<int> list = new () { 1, 2 };

```

Comment:
Instead of creating a List<T> object as a private variable to reduce GC pressure. Also, the 2 lists have to be cleared after each test. I am not sure if the present tests are accurate as you are not clearing the list after each test.